tests/testdata/default-three-rack-three-node-dc.yaml

```diff
@@ -17,18 +17,11 @@ spec:
         resources:
           requests:
             storage: 1Gi
-  resources:
```

Comment:
Why are these removed?

Reply:
I thought some of the initial failures may have been to do the limited resources. The max heap was basically the same as the memory limit specified under `resources`, leaving very little space for off-heap memory.